huggingface/pytorch/training/docker/1.11/py3/cu113/Dockerfile.trcomp.gpu

```diff
@@ -62,7 +62,7 @@ RUN pip install -U --no-cache-dir \
 	
 RUN apt-get update \
  # TODO: Remove upgrade statements once packages are updated in base image
- && apt-get -y upgrade --only-upgrade systemd openssl cryptsetup \
+ && apt-get -y install --only-upgrade systemd openssl cryptsetup \
```

Comment:
using `apt-get upgrade` will upgrade all packages, including libucudnn8 to the latest cu120.